lib/bitcoin.py

```diff
@@ -208,6 +208,7 @@ def rev_hex(s):
 def int_to_hex(i, length=1):
     assert isinstance(i, int)
     s = hex(i)[2:].rstrip('L')
+    s = s.lstrip('0x')
```

Comment:
`hex(i)[2:]` is trying to do the same thing above. what is `i` in your malformed case?